PackageSync.sublime-settings

```diff
@@ -21,7 +21,9 @@
 		"Package Control.ca-list",
 		"Package Control.ca-bundle",
 		"Package Control.system-ca-bundle",
-		"*.sublime-package"
+		"*.sublime-package",
+		"PackageSync.sublime-settings",
```

Comment:
Preferably add a different configuration/flag that would define if the packages are to be included e.g. `sync_package_sync_settings = true`